src/float/float_transceiver/float_transceiver.ino

```diff
@@ -434,7 +434,7 @@ void initRadio() {
 
   if (!rf95.init()) {
     Serial.println("RFM95 radio init failed");
-    while (1) {}
+    while (1);
```

Comment:
Google doesn't like this
![image](https://github.com/user-attachments/assets/cf5a8503-c0a9-4253-a91d-221f7a418f07)